addons/mass_mailing_sms/models/mailing_mailing.py

```diff
@@ -367,3 +369,36 @@ def _get_default_ab_testing_campaign_values(self, values=None):
                 campaign_values['name'] = _("A/B Test: %s", sms_subject)
             campaign_values['ab_testing_sms_winner_selection'] = self.ab_testing_sms_winner_selection
         return campaign_values
+
+    def get_chars_counts(self):
```

Comment:
def ``estimate_extra_character_count`` maybe ? As it does not take into account the content itself, just the extra stuff, if I am not wrong.